app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -81,19 +81,19 @@ class SupplementalApplicationPage extends React.Component {
 
   handleSavePreference = async (preferenceIndex, application) => {
     const { persistedApplication } = this.state
-
-    // We clone the latest saved copy, so we can use the latest saved fields.
     const synchedApplication = cloneDeep(persistedApplication)
 
-    // We use the persisted copy and set only the fields updated in the panel
+    // We need to set the total_monthly_rent in the global application, so we do not overwrite it.
     synchedApplication.total_monthly_rent = application.total_monthly_rent
-    synchedApplication.preferences[preferenceIndex] = application.preferences[preferenceIndex]
```

Comment:
I know @fddayan is out, so he can't answer this right now, but I'm wondering why we removed this line and what the effect is? @akegan @chrisdolendo do you have further insight?

Reply:
my guess is that it may have something to do with the fact that we're refreshing the page on application save now? but I don't know why that wouldn't get rid of the whole synched application thing

Reply:
ok so i tested this and it looks like removing this line introduces a bug if you follow these steps:
1) update a preference in a preference panel (there's no page refresh after updating just a preference)
2) scroll down to bottom of page and click the supp app form Save button
3) the page will refresh, and you will see your changes to the preference undone, because the application was saved with the old preference values
i tried putting this line back in and doing the same test, and with the line in, the above bug behavior doesn't happen, because the supp app form application gets updated with the changes you made to the preference. @fddayan, when you're back online on Monday morning, could you take a look at this and let us know if it's fine for me to put this line back into this PR, or if there was a particular intention behind removing that line?

Reply:
I've chatted with @fddayan and he confirmed that he did not intend to remove this line, so I'm going to put it back in.